### DISCUSSION TITLE
Unable to play media or show thumbnails

### DISCUSSION THREAD

**Discussion by Blaiz0** [Zibbp/ganymede#84] [Q&A] [answered]
Hi,
Really cool project!

I installed it on my NAS, most of the functions seems to be working fine, it is archiving and website is up and running.
But im not able to see thumbnails and media is not playing. 
"The media could not be loaded, either because the server or network failed or because the format is not supported."
The json chat shows up, and im able to play archived files locally from the vods folder.

Could it be a permissions problem?

**Comment by Zibbp** [comment]
Hi,
It sounds like one of the host variables is not set or set incorrectly. Please ensure that the API container's `FRONTEND_HOST` environment variable is set to the IP and port of the Frontend container, like so. `FRONTEND_HOST=http://192.168.1.150:4801`.

Also check to make sure that the frontend container's `API_URL` and `CDN_URL` point to the API IP/port and Nginx IP/port, similar to above.

If this still does not work, please open your browser console and navigate to a page that has missing thumbnails and look for any errors in the console or network tab.

**Comment by Blaiz0** [comment, marked as answer]
Thank you for your reply!

So after a few hours of investigations, i noticed that the nginx log was throwing some errors with a path to /mnt/
Yesterday i changed 
  ganymede-nginx:
  volumes:
 /volume1/docker/ganymede/VODs:/mnt/vods  to /volume1/docker/ganymede/VODs:/vods.  
This broke the nginx.conf. After removing /mnt/ from nginx.conf everything seems to be working 👍 

Had some trouble getting it to compose without errors and synology docker can be a bit wonky.  Couldn't get it to mount anything but the /docker directory. Not sure if i had to change this or if it was just a part of an attempt to get the volumes working.
